OpenRA.Mods.Common/Activities/Air/FlyAttack.cs

```diff
@@ -36,12 +36,12 @@ public FlyAttack(Actor self, Target target)
 		public override Activity Tick(Actor self)
 		{
 			if (!target.IsValidFor(self))
-				return NextActivity;
+				return new ReturnToBase(self);
```

Comment:
But if I call SequenceActivities with ReturnToBase as the last activity will fix this problem?